src/lib/prefix-map.js

```diff
@@ -48,15 +48,19 @@ class PrefixMap {
 
   insert (prefix, item) {
     if (!this.items[prefix]) {
-      this.prefixes.splice(
-        sortedIndex(this.prefixes, prefix), 0, prefix)
+      const index = findIndex(this.prefixes, (e) => { return prefix.length > e.length || prefix > e })
+      if (index === -1) {
+        this.prefixes.push(prefix)
+      } else {
+        this.prefixes.splice(index, 0, prefix)
+      }
     }
     this.items[prefix] = item
     return item
   }
 
   delete (prefix) {
-    const index = sortedIndex(this.prefixes, prefix)
+    const index = indexOf(this.prefixes, prefix)
```

Comment:
Why not use `this.prefixes.indexOf(prefix)` instead of `lodash.indexOf(this.prefixes, prefix)`?